css/style.css

```diff
@@ -0,0 +1,68 @@
+/*................................................
+    HEADER SECTION
+.................................................*/
+
+.container {
+    max-width: 1600px;
+    min-width: 500px;
+    margin: auto;
+    position: fixed;
+    z-index: 1;
+    background-color: white;
+    box-shadow: 0px 0px 30px #8a8585;
+    padding: 8px 30px;
+}
+
+.flex-box {
+    display: flex;
+    justify-content: space-between;  
+}
+
+header {
+    margin: 0 auto;
+}
+
+#logoImage {
+    max-width: 10%;
+}
+
+.flex-box a {
+    color: black;
+}
+.nav-bar {
+    margin-top: 55px;
```

Comment:
since header is a flex box , you can align elements inside without using an fixed margin value